presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorCapabilities.java

```diff
@@ -17,5 +17,5 @@
 {
     NOT_NULL_COLUMN_CONSTRAINT,
     SUPPORTS_REWINDABLE_SPLIT_SOURCE,
-    SUPPORTS_PARTITION_COMMIT,
+    SUPPORTS_TABLE_WRITE_COMMIT,
```

Comment:
As discussed in https://github.com/prestodb/presto/pull/13854#issuecomment-615950267, what about `SUPPORTS_PAGE_SINK_COMMIT`? :)  cc @shixuan-fan